src/ViewModel/Converter/CoverInterviewHighlightItemConverter.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace eLife\Journal\ViewModel\Converter;
+
+use eLife\ApiSdk\Model\Collection;
+use eLife\ApiSdk\Model\Cover;
+use eLife\ApiSdk\Model\Interview;
+use eLife\ApiSdk\Model\Subject;
+use eLife\Journal\Helper\ModelName;
```

Comment:
The "eLife\ApiSdk\Model\Collection;" and "eLife\ApiSdk\Model\Subject;" seem to not be used anymore, so their use statements can be removed.